### PR TITLE
bundling broadcast-channel

### PR DESCRIPTION
**Pull request by jaredperreault-okta**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -471,8 +471,9 @@ Defaults to 300 (five minutes). This is the maximum difference allowed between a
 
 #### `ignoreLifetime`
 
-Token lifetimes are validated using the `maxClockSkew`.
-To override this and disable token lifetime validation, set this value to `true`. 
+> :warning: This option disables token lifetime validation, which can introduce security vulnerability issues. This option should be used for testing purpose. Please handle the error in your own app for production environment.
```
Comment (shuowu-okta) [inline README.md:474]: seems like this change is brought from master, try rebase dev-7 against master and rebase this branch against dev-7

File: scripts/build.js [hunk 2]
```diff
@@ -95,6 +95,9 @@ function removeBuildDir(val) {
   packageJSON[key] = removeBuildDir(packageJSON[key]);
 });
 
+// TODO: testing, included in bundle, do not require installation
+delete packageJSON.dependencies['broadcast-channel'];
```
Comment (shuowu-okta) [inline scripts/build.js:99]: let's move `broadcast-channel` to devDeps, and delete the `devDependencies` field here

**Review by shuowu — APPROVED** [review]

File: rollup.config.js [hunk 3]
```diff
@@ -87,6 +100,11 @@ function createPackageJson(dirName) {
 const getPlugins = (env, entryName) => {
   const outputDir = getOuptutDir(entryName, env);
   let plugins = [
+    commonjs(),
+    nodeResolve({
+      browser: true,
```
Comment (aarongranick-okta) [inline rollup.config.js:105]: should "browser" be set based on the value of "env" ?

**Review by aarongranick-okta — CHANGES_REQUESTED** [review]
Please add a JIRA for removing workarounds for when/if broadcast-channel gets its issues sorted out. Please add comments in the code to indicate where we should remove workarounds.

**Review by aarongranick-okta — APPROVED** [review]

**Pull request closed without merging**